src/main/java/seedu/momentum/model/project/Project.java

```diff
@@ -17,22 +17,28 @@
 
     // Identity fields
     private final Name name;
+    private final Description description;
 
     private final Set<Tag> tags = new HashSet<>();
 
     /**
      * Every field must be present and not null.
      */
-    public Project(Name name, Set<Tag> tags) {
+    public Project(Name name, Description description, Set<Tag> tags) {
         requireAllNonNull(name, tags);
```

Comment:
Is the `requireAllNotNull()` method here supposed to check the description as well?

I believe the description parameter is not optional as of now, right?

Reply:
Yup, will add